src/routes/fields/[id]/index.svelte

```diff
@@ -88,7 +140,7 @@
 				<p>
 					{Math.round(field.area)} ac.
 				</p>
-				<p>Planted: ?</p>
+				<p>Planted: {field.numPlants}</p>
```

Comment:
I'm not sure we need this. I assume this is `plants per acre`? We can keep it as long as it is not required anywhere